### ISSUE TITLE
Be less noisy if everything is correctly formatted

### ISSUE THREAD

**Issue by thomaseizinger** [tbrand/cargo-tomlfmt#13]
Currently, the crate prints:
```
INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
```
if everything properly formatted. In a workspace with several crates, this looks like this:

```
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2019-08-23T04:55:25Z: cargo_tomlfmt: no problem found. good job! :)
```
which is a lot of unnecessary noise.
If everything is properly formatted, the tool should just be silent and exit with 0.

**Comment by tbrand** [comment]
I sent a patch on https://github.com/tbrand/cargo-tomlfmt/pull/14.
It makes the message to be debug level. So, by default, the message isn't shown.

**Comment by tbrand** [comment]
Shipped in v0.1.4. Let me close.
